products/views.py

```diff
@@ -1,3 +1,22 @@
-from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
 
-# Create your views here.
\ No newline at end of file
+from products.models import Menu
+
+class MetaView(View):
+    def get(self, request):
+        menus = Menu.objects.all()
+        
+        results = {}
+        for menu in menus:
+            results[menu.name] = {
+                "menu_id"    : menu.id,
+                "categories" : [
+                    {
+                        "category_id" : category.id,
+                        "name"        : category.name,
+                        "image_url"   : category.image_url
+                    } for category in menu.category_set.all()]
+            }
```

Comment:
menu.name을 key 값으로 잡았는데, 그냥 menu.id랑 같은 위치에 있는 것은 어떤까요?
```suggestion
            results = {
                "menu_id"    : menu.id,
                "menu_name : manu.name,
                "categories" : [
                    {
                        "category_id" : category.id,
                        "name"        : category.name,
                        "image_url"   : category.image_url
                    } for category in menu.category_set.all()]
            } for manu in Menu.objects.all()
```

Reply:
프론트엔드가 구현해놓은게 너무 많아, list로 변경하기가 어려울것 같습니다.